Doc/library/colorsys.rst

```diff
@@ -12,11 +12,12 @@
 
 The :mod:`colorsys` module defines bidirectional conversions of color values
 between colors expressed in the RGB (Red Green Blue) color space used in
-computer monitors and three other coordinate systems: YIQ, HLS (Hue Lightness
-Saturation) and HSV (Hue Saturation Value).  Coordinates in all of these color
-spaces are floating point values.  In the YIQ space, the Y coordinate is between
-0 and 1, but the I and Q coordinates can be positive or negative.  In all other
-spaces, the coordinates are all between 0 and 1.
+computer monitors and four other coordinate systems: YIQ, YUV, HLS (Hue
+Lightness Saturation) and HSV (Hue Saturation Value).  Coordinates in all of
+these color spaces are floating point values.  In the YIQ and YUV space, the
+Y coordinate is between 0 and 1, but the I, Q, U and V coordinates can be
+positive or negative.  In all other spaces, the coordinates are all between
+0 and 1.
```

Comment:
From the BT.709 standard, U should be [-0.436...0.436], and V should be [-0.615...0.615], but since YIQ is using the FCC NTSC, its range in float terms (as opposed to wavelength terms) doesn't seem to be well documented (but it seems to be roughly [-0.5984...0.5984] for I, and [-0.5246..0.5246] for Q).

Since it's kind of varied I figure it's probably best to not bulk these in with this section, but maybe they belong somewhere else (e.g. with the methods?), or if there's a reliable document with these values it might be worth linking to that instead.